_includes/specials/policy/en/05-dos.md

```diff
@@ -0,0 +1,124 @@
+addition to consensus which nodes apply to unconfirmed transactions.
+
+Some policies apply to transactions in relation to one another. A
+previous post discussed an ancestor package limit which also assists
+the block template building algorithm. Bitcoin Core nodes enforce
+symmetrical restrictions on descendant packages. Like block template
+production, the eviction algorithm is more effective when working with
+smaller packages. These two package limits [restrict the computational
+complexity][se descendant limits] of mempool insertion and deletion
+which require updating a transaction’s ancestor and descendant sets.
+
+In the last post, we mentioned the 1sat/vB minimum relay feerate
+policy ("minrelaytxfee" in Bitcoin Core), which sets a “price” for
+network transaction relay. By default, Bitcoin Core nodes do not
+accept any transactions below this feerate. In fact, Bitcoin Core
+nodes do not verify any signatures before checking the feerate
+requirements. A non-mining node doesn’t ever receive fees – they are
+only paid to the miner who confirms the transaction – and some network
+bandwidth has already been used by the time the fee is checked.
+However, fees represent a cost to the attacker. Somebody may send an
+extremely high amount of transactions that require validation and
+relay, but will at some point run out of money to pay the fees.
+
+The Replace by Fee [policy implemented by Bitcoin Core][bitcoin core
+rbf docs] requires that the replacement transaction pay a higher
+feerate than the transaction(s) it directly conflicts with, but also
+requires that it pay a higher total fee. The additional fees (the fees
+paid by the replacement subtracted by the total fees paid by all of
+the transactions it replaces) divided by the replacement transaction's
+virtual size must be at least 1sat/vB. This fee policy is not
+primarily concerned with the incentive compatibility. Rather, it
+prevents repeated replacements of the same transaction in which each
+transaction only pays a small amount more than its predecessor (e.g. 1
+satoshi). In other words, regardless of the feerates of the original
+and replacement transactions, the new transaction must pay "new" fees
+to cover the cost of its own bandwidth at 1sat/vB.
+
+This incremental relay feerate is also used to [raise][pr 6722] a
+node's mempool minimum feerate ("mempoolminfee") when it reaches
+capacity and evicts transactions in descendant score order: the new
+mempool minimum feerate is set to 1sat/vB higher than the descendant
+score of the last transaction (package) evicted, then decays downwards
+towards 1sat/vB at a rate adjusted for how quickly the mempool
+empties. Setting a dynamic mempool minimum feerate does not
+necessarily affect the outcome, as a transaction with a feerate below
+the just-evicted transaction would be evicted immediately after being
+accepted.  However, rejecting the transaction faster affords
+worthwhile savings in validation effort.
+
```

Comment:
I know I'm over 1000 words... my eviction algorithm would kick this paragraph first, as maybe the least interesting example in the post.

Reply:
Yeah, that might be a bit too deep into the weeds, although eviction has been a topic in the past few months that might have people curious.